PublicMapping/districtbuilder
# Style Evaluate Mode panel

## Overview
Design pass of the Evaluate mode panel, button, and the legend over top of the map.

### Checklist


### Demo
<img width="1679" alt="Screen Shot 2021-04-30 at 10 16 18 PM" src="https://user-images.githubusercontent.com/5672295/116737312-24dc3580-aa02-11eb-9c6c-1d1476b15592.png">
<img width="1680" alt="Screen Shot 2021-04-30 at 10 19 39 PM" src="https://user-images.githubusercontent.com/5672295/116737406-3e7d7d00-aa02-11eb-86a2-8863938bccf6.png">
<img width="1680" alt="Screen Shot 2021-04-30 at 10 23 01 PM" src="https://user-images.githubusercontent.com/5672295/116737753-a92eb880-aa02-11eb-884a-e33ede43dba5.png">


### Notes
- I had to set a pixel width for the Evaluate sidebar so it would be the same width as the Districts sidebar... which didn't feel quite right but I couldn't figure out another solution. Open to feedback on this.
- Same ^ for the height of the Evaluate button in order to add the bottom border.
- I added a table heading for the Splits table - "County" and "Split". Not sure if this is right.

## Testing Instructions
- `git pull`
- Click the Evaluate button in top right of the screen of a project.
- Click each box in the summary panel and scroll within the side panel to see how the table performs.
- Note the legend on the map for each box in the summary panel.

~Closes #XXX~
